.travis.yml

```diff
@@ -0,0 +1,9 @@
+
+language: go
+go: 
+ - tip
```

Comment:
@vibe13  Can you confirm if we are using Go 1.6 for productised version of this?